index.js

```diff
@@ -11,8 +11,8 @@ function format(obj) {
     : JSON.stringify(obj);
 }
 
-module.exports = function(hash) {
-  function visit(hash, prefix) {
+module.exports = function(hash, indent=0, newline_after_section=false) {
```

Comment:
Ultimately, whatever new parameter(s) we add here need to be added to the API documentation in `README.md`.

Reply:
I made it an options parameter, empty by default.